scripts/globals/magianobjectives.lua

```diff
@@ -7,647 +7,299 @@
 
 tpz = tpz or {}
 tpz.magian = tpz.magian or {}
+local checks = {} 
 
-local checks = {}
+checks.checkmobkill = function(reqs, params)
+    return reqs.mobid and params.mob and reqs.mobid[params.mob:getID()] and 1 or 0
+end
+
+checks.checkwsonmobsystem = function(reqs, params)
+    return reqs.mobSystem and params.mob and reqs.wSkill and reqs.mobSystem[params.mob:getSystem()] and reqs.wSkill[params.wSkillId] and 1 or 0
+end
+
+checks.checkwskill = function(reqs, params)
+    return reqs.mobSystem and params.mob and reqs.wSkill and reqs.mobSystem[params.mob:getSystem()] and reqs.wSkill[params.wSkillId] and params.mob:isDead() and 1 or 0
+end
```

Comment:
Please name these as camelCase so they're easier to read:
```lua
checkMobKill
checkWsOnMobSystem
checkWsKill 
```
etc.

Reply:
> Content-wise all seems fair 👍
> 
> * Can you rebase and remove `trials.csv` and `trials.xml` from the git tree? I don't know about the history of the repo, but I don't want to introduce any additional large client data-dumps
> * I've left a comment to rename some functions, while you're in there can you make sure things are named and styled consistently?
> 
> ```
> [] that I've tested my code since the last commit in the PR, and will test after any later commits
> ```
> 
> ^ I know a lot of this is data entry, but can you go through and test at least a few of these upgrade steps end-to-end to make sure they're working?
> 
> Thanks :)

Ok, i will test other relics. I'll keep you informed

Reply:
Ok I found an error so I recontrolled everything, now everything is ok.